app/controllers/user/phone-number/get.controller.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+// Local dependencies
+const { renderErrorView } = require('../../../utils/response')
+const userService = require('../../../services/user_service')
+
+module.exports = async (req, res) => {
```

Comment:
method could be named to describe what the code is doing + show a more useful stack trace

Reply:
Would the filename not be in the stack trace? And both these files only export one function so naming it wouldn’t make anything clearer?

Reply:
Yeah, don't think it needs to be strongly required here. Some quick things I can think of: 
* whether something is POST, GET, PATCH impacts what the file name can mean, it's not enforced in the naming structure
* something like `updateUserPhonePageHandler` might read clearer to someone that doesn't understand how everything fits together 
* for this simpler page this is straight forward but I think it's a good practice for more complex pages